ray-operator/controllers/ray/utils/constant.go

```diff
@@ -148,15 +148,15 @@ const (
 	LOCAL_HOST = "127.0.0.1"
 	// Ray FT default readiness probe values
 	DefaultReadinessProbeInitialDelaySeconds = 10
-	DefaultReadinessProbeTimeoutSeconds      = 1
+	DefaultReadinessProbeTimeoutSeconds      = 2
```

Comment:
I ran into issues with the probe timeout in v1.1 as well. 

I am thinking this probe timeout should actually be 4 or 5 seconds for the Head pod. This is because the probe for head pod runs both the agent heath check and GCS health check:
```
wget -T 2 -q -O- http://localhost:52365/api/local_raylet_healthz | grep success
 && wget -T 2 -q -O- http://localhost:8265/api/gcs_healthz | grep success
```  

Which is collectively up to 4 seconds. Thoughts @kevin85421 @HarshAgarwal11

Reply:
Yes I should be 4 to 5 seconds. Because of the OR statement, timeout might get add up. And with 2 sec I was still getting some timeouts, not as frequest as earlier. But after changing it to 5 seconds, I didn't see any timeouts, there were some failures though.

Reply:
Opened https://github.com/ray-project/kuberay/pull/2353

Reply:
Opened a separete issue to track exec probe issues: https://github.com/ray-project/kuberay/issues/2355